test/jdk/java/lang/management/OperatingSystemMXBean/GetSystemLoadAverage.java

```diff
@@ -54,25 +58,25 @@ public class GetSystemLoadAverage {
     // Allow some delta.
     private static double DELTA = 0.05;
 
-    public static void main(String args[]) throws Exception {
-        if (args.length > 1)  {
-            throw new IllegalArgumentException("Unexpected number of args " + args.length);
-        }
+    private static final String OS = System.getProperty("os.name");
+
+    @BeforeMethod
+    void beforeMethod() throws InterruptedException {
+        System.out.println("Wait for 5 seconds.");
+        Thread.sleep(5000);
+    }
 
-        if (args.length == 0) {
+    @Test(invocationCount = 5, timeOut = 300)
```

Comment:
I am not sure it is the correct replacement. Accordingly to TestNG doc the invocationCount = 5 means that TestNG calls the test 5 times.  And test fails if any of the invocations fail while the bash script makes 5 attempts and passes if testcase passed in any of them.

Reply:
Good catch! I've missed the exit statement in the original shell script.